hastexo/hastexo.py

```diff
@@ -110,6 +110,11 @@ class HastexoXBlock(XBlock,
         help="Set the progress check button label. "
              "For example: \"Submit Answer\" or \"Check Progress\"(Default)."
     )
+    show_hints_on_error = Boolean(
+        default=True,
+        scope=Scope.settings,
+        help="Display the tests standard error output stream as a hint"
```

Comment:
Nit: "On progress check failure, display the tests' standard error streams as hints"

Reply:
Done.